docs/public/worldview/index.html

```diff
@@ -12,6 +12,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta name="theme-color" content="#000000" />
     <title>Worldview</title>
+    <link rel='shortcut icon' type='image/x-icon' href='favicon.ico' />
```

Comment:
Favicon didn't work before since we're in a subdirectory. Have to explicitly add it.